deepchecks/suites.py

```diff
@@ -30,5 +30,6 @@
     'train_test_leakage',
```

Comment:
Should remove single_dataset_integrity from here

Reply:
I didn't remove it on purpose, in order to have the deprecation message